src/form.js

```diff
@@ -134,6 +134,8 @@ define([
         this._updateHashtags();
 
         this.tree = new Tree('data', 'control');
+        // initalize #form as /data
+        // if data is not root node, it will be changed later
```

Comment:
I think there may already be one, although it may not be an explicit test. I'll do some digging later to find one/make a new one